hawc/apps/lit/models.py

```diff
@@ -707,6 +707,11 @@ def existing_doi_map(cls, dois: List[str]) -> Dict[str, int]:
             ).values_list("unique_id", "id")
         }
 
+    def save(self, *args, **kwargs):
```

Comment:
we dont need this anymore right? i guess we can keep it b/c it doesnt hurt, but it does seem redundant

Reply:
Even though it's redundant, I like having it there in case a bug is introduced elsewhere, that way we don't end up creating any duplicate dois in the db (where one is lowercase and one is uppercase)

Reply:
sounds good, let's keep it in